GETTING_STARTED.md

```diff
@@ -43,6 +24,25 @@ Fetch the latest version of `master` from `upstream` (ie. the main repo):
 
     git fetch upstream master
 
+### Instalation
+
+This project needs specific ruby/bundler versions as well as node/yarn specific versions. For a local setup you will need:
+
+* Install or change your Ruby version according to the one specified at [.ruby-version](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.ruby-version) file)
+    - To manage versions, it's recommended to use [rbenv](https://github.com/rbenv/rbenv) or [RVM](https://rvm.io/)
```

Comment:
our setup script uses `rbenv` to try install the correct version of ruby, I would remove mention of `RVM` here.

Reply:
Agreed, although you could use RVM or other method, we don't need to suggest it or try to support it.